contracts/ColonyTask.sol

```diff
@@ -420,12 +386,23 @@ contract ColonyTask is ColonyStorage {
     emit TaskDeliverableSubmitted(_id, _deliverableHash);
   }
 
+  function submitTaskDeliverableAndRating(uint256 _id, bytes32 _deliverableHash, bytes32 _ratingSecret) public
+  stoppable
+  {
+    submitTaskDeliverable(_id, _deliverableHash);
+    submitTaskWorkRating(_id, 0, _ratingSecret);
+  }
+
   function finalizeTask(uint256 _id) public
   stoppable
   taskExists(_id)
-  taskWorkRatingsAssigned(_id)
+  canFinalizeTask(_id)
```

Comment:
Rename `canFinalizeTask` to `taskWorkRatingsComplete`? 
canFinalize implies potentially the other modifier rules we have in there, e.g. task exists and task not already finalized

Reply:
The associated error message `colony-task-cannot-finalize` can also be improved to `colony-task-ratings-incomplete`